### PR TITLE
DPAD-749: [mobile] make player "sticky" as you scroll on canonical pages

### PR DESCRIPTION
**Pull request by gundams-and-dragons**
https://fandom.atlassian.net/browse/DPAD-749

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fandom/jwplayer-fandom",
-	"version": "2.0.0-beta-12",
+	"version": "2.0.0-beta-12-dpad-749-2",
```
Comment (gundams-and-dragons) [inline package.json:3]: I'll change this to the next version before merge

File: src/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx [hunk 2]
```diff
@@ -1,12 +1,51 @@
...
+const CanonicalVideoPlayer: React.FC<CanonicalVideoPlayerProps> = ({ currentVideo, onComplete, isMobile }) => {
+	const ref = useRef<HTMLDivElement>(null);
+	const onScreen = useOnScreen(ref, '0px', 1);
+	const isScrollPlayer = isMobile && !onScreen;
 
-const CanonicalVideoPlayer: React.FC<CanonicalVideoPlayerProps> = ({ currentVideo, onComplete }) => {
 	return (
-		<PlayerWrapper playerName={'base-player'}>
-			<VideoPlayer currentVideo={currentVideo} onComplete={onComplete} />
+		<PlayerWrapper playerName="canonical-video-player">
```
Comment (p-forfa) [inline src/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx:43]: 👍

File: test-jw/src/App.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
-import { VideoPlayer } from './bundle.esm.js'
-import { WIREWAX_VIDEO, REGULAR_VIDEO, ARTICLE_VIDEO_DETAILS } from './videoConfigs';
+import { CanonicalVideoPlayer } from './bundle.esm.js'
+import { WIREWAX_VIDEO, CANONICAL_VIDEO, ARTICLE_VIDEO_DETAILS } from './videoConfigs';
 
 function App() {
   return (
     <div className="App">
-		<VideoPlayer playlist={REGULAR_VIDEO ?? WIREWAX_VIDEO} />
+		  <CanonicalVideoPlayer currentVideo={CANONICAL_VIDEO ?? WIREWAX_VIDEO} />
```
Comment (p-forfa) [inline test-jw/src/App.js:7]: Good to have this test App setup. Might be useful in the future to do some conditional here for all the video players we have, but not right now.

**Review by p-forfa — APPROVED** [review]
LGTM

**Pull request merged**